src/program/command_mode.rs

```diff
@@ -327,9 +327,12 @@ impl<'a> Program<'a> {
                 }
                 self.newglob(&arguments);
             }
-            Commands::Help => {
-                self.ui_state.render_help = !self.ui_state.render_help;
-            }
+            Commands::Help => match self.ui_state.render_help {
+                HelpRender::None | HelpRender::Normal => {
```

Comment:
Would this be expressed clearly with `_`

If the HelpRender is `Command`. Set it to None
Else Render the `Command` helpbox
_